_includes/partials/homePage-integrations.html

```diff
@@ -9,10 +9,11 @@ <h2>
           </p><p style="margin:2vw 10vw 0vw 10vw;">
             Meshery's Kubernetes-native approach means you can easily incorporate Meshery into your existing workflow without additional setup or integration effort.
           </p>
-        <div class="integrations-3d-link">
-            <a class="highlight" href="/integrations"
-                style="display: block; margin: max(3vh,3vw) auto 0; width: fit-content;">See All Integrations</a>
-        </div>
+    </div>
+    <div id="iconContainer" class="integrations-container container" style="column-gap:max(2vh,2vw);row-gap:1.5rem;"></div>
+    <div class="integrations-3d-link">
+        <a class="highlight" href="/integrations"
+            style="display: block; margin: max(2vh,2vw) auto 0; width: fit-content;">See All Integrations</a>
```

Comment:
I wonder why we are giving `inline` css, instead, can we write css in external css file? 
Or we have written to maintain the `overall` code consistency in whole codebase? 
As it is least preferrable to use inline css and considered as bad practice. 
Your inputs? @thisiskaransgit ?

Reply:
It was written like this already, so I guess it has something to do with consistency.